api/src/modules/sourcing-locations/sourcing-location.entity.ts

```diff
@@ -136,6 +158,7 @@ export class SourcingLocation extends TimestampedBaseEntity {
     (supplier: Supplier) => supplier.sourcingLocations,
     {
       eager: false,
+      onDelete: 'CASCADE',
```

Comment:
and here

Reply:
[Same reason here](https://github.com/Vizzuality/landgriffon/pull/76#discussion_r679136595)